evaluate.py

```diff
@@ -39,31 +39,31 @@ def evaluate(config, model, dataset_loader, eval_metric, split='dev', dump=True)
 
 
 class FewShotMetrics():
-    def __init__(self, config, dataset, few_shot_limit=5, target_label='label'):
+    def __init__(self, config, dataset=None, few_shot_limit=5, target_label='label'):
         # read train / test labels
         # dev is not considered for now
-        test_labels = np.hstack([instance[target_label]
-                                 for instance in dataset['test']])
-        train_labels = np.hstack([instance[target_label]
-                                  for instance in dataset['train']])
+        # test_labels = np.hstack([instance[target_label]
+                                 # for instance in dataset['test']])
+        # train_labels = np.hstack([instance[target_label]
+                                  # for instance in dataset['train']])
 
         self.config = config
-        self.class_num = dataset['train'].num_class
+        self.class_num = config.num_class
```

Comment:
class_num vs.  num_class not good
how about num_classes in all places?

Reply:
Solved. Eleven is working on a refactoring of evaluate.py.